docs/source/app_developers_guide/ubuntu.rst

```diff
@@ -47,33 +59,37 @@ test networks. In this case, you need to create a genesis block when
 instantiating a new network.
 
 The genesis block contains some initial values that are necessary when a
-Sawtooth distributed ledger is created and used for the first time.
+Sawtooth distributed ledger is created and used for the first time. 
+One of the settings in the genesis block that should be set is the 
+key that is authorized to set and change configuration settings, as 
+shown below using the **sawtooth config genesis** command. 
 
 To create the genesis block, log in to the development environment CLI and run
 the following command:
 
 .. code-block:: console
 
-  $ sawtooth admin genesis
-  Generating /home/ubuntu/sawtooth/data/genesis.batch
+  $ sawtooth keygen my_key
```

Comment:
Omit my_key and the -k option on config genesis below.  This is kind of advanced usage, but here we are introducing the sequence, so we should simplify it.